v2.1/admin-ui-hardware-dashboard.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Hardware Dashboard
+summary: The Hardware dashboard lets you monitor CPU usage, disk throughput, network traffic, storage capacity, and memory.
+toc: true
+---
+
+The **Hardware** dashboard lets you monitor CPU usage, disk throughput, network traffic, storage capacity, and memory. To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** on the left-hand navigation bar, and then select **Dashboard** > **Hardware**.
+
+The **Hardware** dashboard displays the following time series graphs:
+
+## User CPU Percent
+
+<img src="{{ 'images/v2.1/admin_ui_user_cpu.png' | relative_url }}" alt="CockroachDB Admin UI User CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the User CPU Percent graph shows the percentage of the total CPU seconds per second used by the CockroachDB process for the selected node.
+
+- In the cluster view, the graph shows the percentage of the total CPU seconds per second used by the CockroachDB process across all nodes.
+
+{{site.data.alerts.callout_info}}
+For multi-core systems, the User CPU Percent can be greater than 100%. Full utilization of one core is considered as 100% CPU usage. If you have n cores, then the User CPU Percent can range from 0% (indicating an idle system) to (n*100)% (indicating full utilization).
+{{site.data.alerts.end}}
+
+## System CPU Percent
+
+<img src="{{ 'images/v2.1/admin_ui_system_cpu.png' | relative_url }}" alt="CockroachDB Admin UI System CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the percentage of the total CPU seconds per second used by the system calls made by CockroachDB for the selected node.
+
+- In the cluster view, the graph shows the percentage of the total CPU seconds per second used by the system calls made by CockroachDB across all nodes.
+
+{{site.data.alerts.callout_info}}
+For multi-core systems, the System CPU Percent can be greater than 100%. Full utilization of one core is considered as 100% CPU usage. If you have n cores, then the User CPU Percent can range from 0% (indicating an idle system) to (n*100)% (indicating full utilization).
+{{site.data.alerts.end}}
+
+## Memory Usage
+
+<img src="{{ 'images/v2.1/admin_ui_memory_usage_new.png' | relative_url }}" alt="CockroachDB Admin UI Memory Usage graph" style="border:1px solid #eee;max-width:100%" />
+
+- In the node view, the graph shows the memory in use for the selected node.
+
+- In the cluster view, the graph shows the memory in use across all nodes in the cluster.
+
+## Disk Read Bytes
```

Comment:
please add a note for
- disk {read, write, iops}
- net {read, write}

that those metrics are across the whole host, not just for the cockroach process, i.e. they could show disk activity from other processes. Memory and CPU, on the other hand, are just for the cockroach process. Kind of confusing, but I wasn't able to find a good way to get the per-process IO info on both Mac and Linux.

Reply:
@vilterp I made the changes - might have made it worse. Can you check and let me know?